cmd/libs/go2idl/deepcopy-gen/generators/deepcopy.go

```diff
@@ -84,38 +137,61 @@ func Packages(context *generator.Context, arguments *args.GeneratorArgs) generat
 		}
 	}
 
-	for _, p := range context.Universe {
-		// Short-circuit if this package has not opted in.
-		if tagvals := types.ExtractCommentTags("+", p.Comments)["k8s:deepcopy-gen"]; tagvals == nil {
-			// No tag for this package.
-			continue
-		} else if tagvals[0] != "generate" && tagvals[0] != "register" {
-			// Unknown tag.
-			glog.Errorf("Uknown tag value '+k8s:deepcopy-gen=%s'", tagvals[0])
+	for i := range inputs {
```

Comment:
>
If you didn't list the pkg as an input, we will not process it at all. Full stop. So if even ONE type wants generation, you have to list the pkg as an input, which will trigger here

Yes - that one I agree with.

What I'm worried about is that we won't generate conversions at all that:
- doesn't have doc.go comment 
- have opt-in for generation by setting the type comment
Or am I missing something?